enderio-base/src/main/resources/assets/enderio/config/recipes/base.xml

```diff
@@ -693,7 +693,87 @@ XML editor will display that as tooltips when editing this file.
       <output name="ingotSteel"/>
     </alloying>
   </recipe>
-
+ 
+  <recipe name="Alloy: FerroBoron (Ingots) (Nuclearcraft)" required="false">
+    <alloying energy="4000" exp="0">
+      <input name="ingotSteel"/>
+      <input name="ingotBoron"/>
+      <output name="ingotFerroboron" amount="2"/>
+    </alloying>
+  </recipe>
+ 
+  <recipe name="Alloy: FerroBoron (Dusts) (Nuclearcraft)" required="false">
+    <alloying energy="4000" exp="0">
+      <input name="dustSteel"/>
```

Comment:
We don't do dust recipes in addition to ingot alloying recipes because that interferes with the dust->ingot smelting.